lerna.json

```diff
@@ -3,7 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.5.0",
+  "version": "independent",
```

Comment:
This means that each component will update its version independently?

Reply:
correct